python/chigger/RenderWindow.py

```diff
@@ -262,7 +262,7 @@ def write(self, filename):
         if ext not in writers:
             w = ', '.join(writers.keys())
             msg = "The filename must end with one of the following extensions: {}.".format(w)
-            mooseutils.mooseError(msg)
+            mooseutils.mooseError(msg, dialog=True)
```

Comment:
Can we add a dialog=False to the RenderWindow.write method instead. Then in peacock the call to this could pass True? I don't want get a dialog when running just a chigger script.